### PR TITLE
Only prod shall have prod vars

### PR DESCRIPTION
**Pull request by andrew-jameson**
## Summary of Changes
Please see #2334. Changed direction after having conversations about path forward.

### CODE REVIEW

**Comment by andrew-jameson** [comment]
Merge notify test

**Comment by andrew-jameson** [comment]
notify test

**Comment by andrew-jameson** [comment]
Test 3

**Comment by andrew-jameson** [comment]
https://circleci.com/docs/github-integration/

**Review by n0remac — APPROVED** [review]

File: scripts/deploy-backend.sh [hunk 1]
```diff
@@ -47,39 +47,22 @@ set_cf_envs()
...
-  "PROD_JWT_KEY"
   "LOGGING_LEVEL"
-  "PROD_ACR_VALUES"
-  "PROD_OIDC_OP_AUTHORIZATION_ENDPOINT"
-  "PROD_CLIENT_ASSERTION_TYPE"
-  "PROD_OIDC_RP_CLIENT_ID"
-  "PROD_OIDC_OP_ISSUER"
-  "PROD_OIDC_OP_JWKS_ENDPOINT"
-  "PROD_OIDC_OP_LOGOUT_ENDPOINT"
-  "PROD_OIDC_OP_TOKEN_ENDPOINT"
```
Comment (raftmsohani) [inline scripts/deploy-backend.sh:60]: I don't understand, what happens when deploying to prod then?

Comment (andrew-jameson) [reply scripts/deploy-backend.sh:60]: Our CircleCI won't be deploying to prod. In the HHS CircleCI, these variables do not have the prefix and will deploy to prod as we had previously agreed on.

**Review by raftmsohani — APPROVED** [review]
LGTM!

**Pull request merged**